src/main/java/uk/gov/hmcts/reform/civil/model/RepaymentPlanLRspec.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.civil.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class RepaymentPlanLRspec {
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING)
+    private final BigDecimal paymentAmount;
+    private final String repaymentFrequency;
+    private final double repaymentLength;
```

Comment:
Let's omit this repaymentLength field because we are not building it yet

Reply:
change incorporated